pkg/network/tracer/tracer_windows.go

```diff
@@ -214,37 +214,20 @@ func (t *Tracer) getConnTelemetry() map[network.ConnTelemetryType]int64 {
 	//   dns   ( the dns handle stats)
 	//   each of the strings in DriverExpvarNames.  We're interested
 	//   in driver.flowHandleStats, which is "driver_flow_handle_stats"
-	if allstats, err := t.driverInterface.GetStats(); err == nil {
-		if flowStats, ok := allstats["driver_flow_handle_stats"].(map[string]int64); ok {
-			if fme, ok := flowStats["num_flows_missed_max_exceeded"]; ok {
```

Comment:
this stat was never being populated. removed it at the direction of the windows team